### PR TITLE
aws2-wrap 1.2.8 (new formula)

### PR DESCRIPTION
**Pull request by nitrocode**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

```
FORMULA=Formula/aws2-wrap.rb
brew install --build-from-source $FORMULA
brew test $FORMULA
brew audit --strict $FORMULA
brew audit --new $FORMULA
```

### CODE REVIEW

File: Formula/aws2-wrap.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Aws2Wrap < Formula
+  include Language::Python::Virtualenv
+
+  desc "Script to export current AWS SSO credentials or run a sub-process with them"
+  homepage "https://github.com/linaro-its/aws2-wrap"
+  url "https://files.pythonhosted.org/packages/fa/07/b0fbfc6d3640d0a55250b26900c534f655046bbbf081d111eab95c6611c7/aws2-wrap-1.2.8.tar.gz"
+  sha256 "3e39be94c10e700a388fdc35da59a9232e766d65f05e57cd00651082a9887346"
+  license "GPL-3.0-only"
+
+  depends_on "python@3.9"
```
Comment (chenrui333) [inline Formula/aws2-wrap.rb:10]:
```suggestion
  depends_on "python@3.10"
```

**Review by chenrui333 — DISMISSED** [review]

**Review by chenrui333 — DISMISSED** [review]

File: Formula/aws2-wrap.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class Aws2Wrap < Formula
+  include Language::Python::Virtualenv
+
+  desc "Script to export current AWS SSO credentials or run a sub-process with them"
+  homepage "https://github.com/linaro-its/aws2-wrap"
+  url "https://files.pythonhosted.org/packages/fa/07/b0fbfc6d3640d0a55250b26900c534f655046bbbf081d111eab95c6611c7/aws2-wrap-1.2.8.tar.gz"
+  sha256 "3e39be94c10e700a388fdc35da59a9232e766d65f05e57cd00651082a9887346"
+  license "GPL-3.0"
```
Comment (SMillerDev) [inline Formula/aws2-wrap.rb:8]:
```suggestion
  license "GPL-3.0-only"
```

**Comment by nitrocode** [comment]
@chenrui333 not sure why the license was updated ? I reverted the change that @SMillerDev suggested and force pushed.

**Review by chenrui333 — DISMISSED** [review]

**Comment by nitrocode** [comment]
Looks like the github actions stuck in a queued state. Is it possible to rerun them or is it better to force push?

**Comment by SMillerDev** [comment]
You'll need to rebase the branch.

**Comment by nitrocode** [comment]
I ran `git commit --amend --no-edit --allow-empty && git push -f` because the checks failed due to some github action runner issue

**Comment by nitrocode** [comment]
Thanks @SMillerDev. I did a pull and rebased from the upstream repo's default branch.

**Comment by nitrocode** [comment]
Hmm still running into a failure `CI / tests (12-2701143437-1) (pull_request)` but I'm unsure the reason this is failing

https://github.com/Homebrew/homebrew-core/runs/7419480594?check_suite_focus=true

File: Formula/aws2-wrap.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    mkdir testpath/".aws"
+    touch testpath/".aws/config"
+    ENV["AWS_CONFIG_FILE"] = testpath/".aws/config"
+    assert_match "Cannot find profile 'default'",
+      shell_output("#{bin}/aws2-wrap 2>&1", 1).strip
```
Comment (carlocab) [inline Formula/aws2-wrap.rb:26]: Splitting this line doesn't seem necessary at all.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**